File: Capítulo_3/Iterando pelos elementos de arrays bidimensionais/Código em Python/iteracaoMatriz.py
def printMatrix(array):
    for i in range(len(array)):
        row = ""
        for j in range(len(array[i])):
            row += str(array[i][j]) + " "
        print(row)

averageTemp = []
averageTemp.append([72, 75, 79, 79, 81, 81])
averageTemp.append([81, 79, 75, 75, 73, 73])

printMatrix(averageTemp)

File: Capítulo_3/Iterando pelos elementos de arrays bidimensionais/iteracaoMatriz.js
export function printMatrix(array) {
    for (let i = 0; i < array.length; i++) {
        let row = "";
        for (let j = 0; j < array[i].length; j++) {
            row += array[i][j] + " ";
        }
        console.log(row)
    }
}
